waiter/src/waiter/token.clj

```diff
@@ -246,6 +246,18 @@
     [kv-store]
     (into {} (kv/fetch kv-store token-owners-key)))
 
+  (defn get-token-index
+    "Given a token, determine the correct index entry of the token. Returns nil if token doesn't exist."
+    [kv-store token & {:keys [include refresh] :or {include #{} refresh false}}]
```

Comment:
```suggestion
    [kv-store token & {:keys [include refresh] :or {include #{:token :owner} refresh false}}]
```
Keeps the changes backward compatible with previous function.

Reply:
An index, by default, contains the `owner` and `token`. Not sure what we would label something that doesn't have these.
Maybe drop making `include` configurable?